src/FamilyHubs.Referral.Core/Models/PartialTextBoxViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace FamilyHubs.Referral.Core.Models;
+
+public class PartialTextBoxViewModel
+{
+    public string HeadingText { get; set; } = default!;
```

Comment:
It would be better to remove the initialisations to `default!` and either have the properties get only with a ctor, or have them as get & init.
Having a property non-nullable, but then setting them to null, should only really be used as a workaround when it's necessary.

Reply:
Uses required key word